python_modules/libraries/dagster-twilio/dagster_twilio_tests/test_resources.py

```diff
@@ -29,7 +33,7 @@ def twilio_solid(context):
         assert "The 'To' number +15005550001 is not a valid phone number" in str(exc_info.value)
 
     result = execute_solid(
-        twilio_solid,
+        twilio_op,
```

Comment:
up to you, but could be worth it to wrap `twilio_op` in a job so we can use `execute_in_process` instead of `execute_solid` and bring this test file out of the solid-age